pkg/api-server/read_only_resource_endpoints_test.go

```diff
@@ -83,6 +85,13 @@ var _ = Describe("Read only Resource Endpoints", func() {
 
 			// then
 			Expect(response.StatusCode).To(Equal(405))
+			body, err := ioutil.ReadAll(response.Body)
+
+			Expect(err).To(BeNil())
```

Comment:
Overall to check if error not occurred we use `Expect(err).ToNot(HaveOccured())`